modules/issue_tracker/help/issue_tracker.md

```diff
@@ -1,19 +1,11 @@
-# Issue Tracker Help
+# Issue Tracker
 
-The Issue Tracker module allows users to submit, edit, view and track issues 
+This module allows you to submit, edit, view, and track issues.
 
-## View Issues
+Use the *Selection Filter* section to search for specific issues, if desired. In the data table, you can navigate between different tabs—**All Issues**, **Active Issues**, **Closed Issues**, and **My Issues**—to further narrow your results. 
 
-Click on a tab for easy access to pre-filtered issue lists. Use the drop-down fields to further sort the data. The date field allows you to specify the earliest submitted issue you would like to see as well as issues submitted not past a certain date. 
+In the table, in the *Title* column, you can click on any issue name to edit its details. 
 
-## Edit Issue
+Click **New Issue** if you wish to add a new issue. Then, populate the fields and click **Submit Issue**.
 
-Click on the blue issue title within the table to view and edit further details for that issue. If you do not have edit permissions you will be blocked from doing so but will still be able to view the data. 
-
-## Add issue
-
-To add an issue, click “add issue”. After valid data is entered (users must supply a title, a priority and cannot enter incorrect PSCIDs or visit labels), you will be redirected back to the default table view. 
-
-## Permissions
-
-Permissions are in accordance with the data permissions granted to that user - if a user can not see data outside of their site, they will only be able to view issues relevant to their site. If a user has DCC permission they will be able to see issues relevant to all sites. Additionally, users can be designated reporters or developers. Reporters can add issues, edit their own issues, and comment on all issues. Developers can additionally edit all issues, and mark them as resolved. Most users of Loris will be designated as reporters.  
+Note that you can add users to the "Watching" list - this will send email notifications to the selected users when the issue is created and updated. If you are the author of an issue, and you add yourself as Watching, you won't be sent notifications about your own updates. 
```

Comment:
>  If you are the author of an issue, and you add yourself as Watching, you won't be sent notifications about your own updates. 

Can someone verify if this is true? It sounds counter-intuitive to me.

Reply:
Derek verified it during 22 testing, with me.  

It's the same way Redmine works - when you update a ticket you don't get an email about it (too spammy). You only get notified if others update the ticket.